From: Jakub Kicinski <jakub.kicinski@netronome.com>
Date: Mon, 17 Oct 2016 18:02:22 +0100
Subject: net: report right mtu value in error message
Patch-mainline: v4.10-rc1
Git-commit: a0e65de71527477b100afe1e9ab2667d70f94d9e
References: bsc#1033962

Check is for max_mtu but message reports min_mtu.

Signed-off-by: Jakub Kicinski <jakub.kicinski@netronome.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Benjamin Poirier <bpoirier@suse.com>
---
 net/core/dev.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/net/core/dev.c
+++ b/net/core/dev.c
@@ -6181,7 +6181,7 @@ int dev_set_mtu(struct net_device *dev,
 
 	if (dev->max_mtu > 0 && new_mtu > dev->max_mtu) {
 		net_err_ratelimited("%s: Invalid MTU %d requested, hw max %d\n",
-				    dev->name, new_mtu, dev->min_mtu);
+				    dev->name, new_mtu, dev->max_mtu);
 		return -EINVAL;
 	}
 
